contracts/src/validity/OPSuccinctDisputeGame.sol

```diff
@@ -152,4 +152,14 @@ contract OPSuccinctDisputeGame is ISemver, Clone, IDisputeGame {
         rootClaim_ = rootClaim();
         extraData_ = extraData();
     }
+
+    ////////////////////////////////////////////////////////////////
+    //                     IMMUTABLE GETTERS                      //
+    ////////////////////////////////////////////////////////////////
+
+    /// @notice Getter for the L2OutputOracle.
+    /// @return l2OutputOracle_ The address of the L2OutputOracle.
+    function l2OutputOracle() external view returns (address l2OutputOracle_) {
+        l2OutputOracle_ = L2_OUTPUT_ORACLE;
+    }
```

Comment:
Do we need this? Isn't `L2_OUTPUT_ORACLE` as an immutable address/variable already viewable?

Reply:
It's internal variable so the call to view will revert.

```solidity
address internal immutable L2_OUTPUT_ORACLE;
```

Reply:
Sg